enterprise/models.py

```diff
@@ -850,13 +858,31 @@ class EnterpriseCustomerCatalog(TimeStampedModel):
             "included in the catalog."
         )
     )
+    enabled_course_modes = JSONField(
+        default=mode_slugs,
+        blank=True,
+        null=True,
+        load_kwargs={'object_pairs_hook': collections.OrderedDict},
+        help_text=_("Course modes that will be displayed on the Enterprise course enrollment landing page.")
+    )
+
     history = HistoricalRecords()
 
     class Meta(object):
         verbose_name = _("Enterprise Customer Catalog")
         verbose_name_plural = _("Enterprise Customer Catalogs")
         app_label = 'enterprise'
 
+    def __init__(self, *args, **kwargs):
+        """
+        Initialize the object.
+        """
+        super(EnterpriseCustomerCatalog, self).__init__(*args, **kwargs)
+        try:
+            self.enabled_course_modes = json.loads(self.enabled_course_modes)
+        except (ValueError, TypeError):
+            pass
```

Comment:
Can this be done in `clean` or something?

Reply:
You don't need to do this at all. `JSONField` will handle the conversion from JSON to Python for you.